src/porepy/fracs/fractures_2d.py

```diff
@@ -531,6 +533,31 @@ def _insert_auxiliary_points(
         self._decomposition["edges"] = lines
         self._decomposition["mesh_size"] = mesh_size
 
+    @pp.time_logger(sections=module_sections)
+    def _set_dfn_mesh_size(
+        self,
+        mesh_size_frac: Optional[float]=None,
+        mesh_size_bound: Optional[float]=None,
+        mesh_size_min: Optional[float]=None,
```

Comment:
Is mesh_size_min outdated? I can not find that it is ever used (i.e., in pp.fracs.tools.determine_mesh_size(..))

Reply:
It is used, L48 in pp.fracs.tools.

Reply:
It is assigned to val_min, but val_min is not used as far as I can tell